Maple2.File.Ingest/Mapper/ItemMapper.cs

```diff
@@ -70,6 +72,73 @@ public class ItemMapper : TypeMapper<ItemMetadata> {
                 expirationDuration = data.life.usePeriod;
             }
 
+            var hairList = new List<DefaultHairMetadata>();
+            // parse default hair positions
+            foreach (Slot slot in data.slots.slot.Where(dataSlots => dataSlots.name == "HR")) {
+
+                // not sure what the difference/significance is within the multiple scale entries. Currently just using the first one
+                float minScale = slot.scale.ElementAtOrDefault(0)?.min ?? 0f;
+                float maxScale = slot.scale.ElementAtOrDefault(0)?.max ?? 0f;
+                switch (slot.asset.Count) {
+                    case 3: // Hair has front and back positionable hair section
+                        var backHairPositions3 = new Vector3[slot.asset[1].custom.Count];
```

Comment:
we don't need this array if you only ever access the current index

Reply:
Fixed! I kept the array because we'll need all entries.

Reply:
still seems unnecessary, how is this different from:

```cs
for (int index = 0; index < slot.asset[1].custom.Count; index++) {
    hairList.Add(new DefaultHairMetadata(
        BackPosition: slot.asset[1].custom[index].position,
        BackRotation: slot.asset[1].custom[index].rotation,
        FrontPosition: slot.asset[2].custom[index].position,
        FrontRotation: slot.asset[2].custom[index].rotation,
        MinScale: minScale,
        MaxScale: maxScale));
}
```

Reply:
oh.. yeah true